src/ExpenseTrackerGrupo2/Services/ExpenseService.cs

```diff
@@ -0,0 +1,89 @@
+using ExpenseTrackerGrupo2.Business.Services.Interfaces;
+using ExpenseTrackerGrupo2.Business.Services.Mappers.Requests.Expenses;
+using ExpenseTrackerGrupo2.DataAccess.Concretes;
+using ExpenseTrackerGrupo2.DataAccess.Entities;
+
+public class ExpenseService : IExpenseService
+{
+    private readonly ExpenseRepository _expenseRepository;
+
+    public ExpenseService(ExpenseRepository expenseRepository)
+    {
+        this._expenseRepository = expenseRepository;
```

Comment:
I suggest you to delete `this` word. I know we can use it, but at least in our code conventions of Programming 6 we didn't use it.